routes/data.js

```diff
@@ -203,6 +203,48 @@ exports.organizationMemberGet = async function (req, res) {
   res.send(memberInfo.data());
 };
 
+exports.getMember = async function (req, res) {
+  const memberData = req.user;
+  const userData = {
+    authenticationID: memberData.id,
+    name: memberData.displayName,
+  };
+  const memberSnapshot = await firestore
+    .collection(resolveCollectionName('Members'))
+    .where('authenticationID', '==', memberData.id);
+
+  memberSnapshot.get().then(function (doc) {
+    if (doc.docs[0]) {
+      res.send(doc.docs[0].id);
+    } else {
+      firestore
+        .collection(resolveCollectionName('Members'))
+        .doc()
+        .set(userData)
+        .then(
+          memberSnapshot.get().then(function (doc) {
+            res.send(doc.docs[0].id);
+          })
+        );
+    }
+  });
+};
```

Comment:
nit: a general notes on the current backend logics. I think we need to add some inline documentations to track details on the implementation. As we are adding more code, we need to make sure the codebase has high readability and maintainability. 

In this block, I think the if/else blocks in line 216-230 might need some documentation on explaining its decision making logic.